### ISSUE TITLE
Responsiveness...

### ISSUE THREAD

**Issue by janvaniperen** [hemangsk/water-ice#1]
Am I correct that it does not render correctly in mobile? Right side seems to scale, but left pane does not.

**Comment by hemangsk** [comment]
@ipears  Yes!  absolutely correct ..it really needs a responsive makeover, especially the left pane issue.
